front/app/containers/Admin/reporting/components/ReportBuilder/Widgets/ChartWidgets/ActiveUsersWidget/ActiveUsersCard.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useRef } from 'react';
+
+// hooks
+import useActiveUsers from 'modules/commercial/analytics/admin/components/ActiveUsersCard/useActiveUsers';
+
+// components
+import { Box, Title } from '@citizenlab/cl2-component-library';
+import Chart from 'modules/commercial/analytics/admin/components/ActiveUsersCard/Chart';
+import Statistic from 'components/admin/Graphs/Statistic';
+import NoChartData from '../NoChartData';
+
+// i18n
+import { useIntl } from 'utils/cl-intl';
+import messages from 'modules/commercial/analytics/admin/components/ActiveUsersCard/messages';
```

Comment:
Suggest removing this reference and adding the message you need to ChartWidgets/messages.ts